File: templates/tests-go/BREACH-005.go
package compliance

import (
	"testing"
)

func TestVerifyMFAEnabled(t *testing.T) {
	config := GetConfig()
	if config.Authentication.MFA != "Enabled" {
		t.Errorf("MFA must be enabled, got %s", config.Authentication.MFA)
	}
}

File: templates/tests-java/BREACH-005.java
package compliance;

import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

class MFATest {
    @Test
    void testVerifyMFAEnabled() {
        Config config = Config.load();
        assertEquals("Enabled", config.getAuthentication().getMfa(),
            "MFA must be enabled");
    }
}
